### PR TITLE
Add Content-Security-Policy for c2c and document them

### PR DESCRIPTION
**Pull request by sbrunner**
https://jira.camptocamp.com/browse/GSGMF-863

### CODE REVIEW

File: doc/integrator/headers.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.. _integrator_headers:
+
+Headers
+=======
+
+We can set with some configuration some HTTP headers on defferent path.
```
Comment (jwkaltz) [inline doc/integrator/headers.rst]: What about:

c2cgeoportal produces in its responses HTTP headers with certain default values. You may wish to override the values being written. To achieve this, you can configure a `global _headers`` section in your vars file with the following syntax:

File: doc/integrator/headers.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+.. code:: yaml
+
+    vars:
+      global_headers:
+        - pattern: <regex>
+          headers:
+            <header>: <value>
+
+If a path match more than one pattern, all the headers will be applyed, if one header applyed tweice
```
Comment (jwkaltz) [inline doc/integrator/headers.rst]: Suggestion:
If a path matches more than one pattern, all headers listed in each match will be applied. If the same header is matched more than once, the last value is kept.

File: doc/integrator/headers.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+the last one win.
+
+For the ``Content-Security-Policy`` we add some vars to be customisable.
```
Comment (jwkaltz) [inline doc/integrator/headers.rst]: Not sure I understand... is it:

For the ``Content-Security-Policy`` headers in particular, c2cgeoportal includes specific variables in its standard template, to facilitate the customization of these values.

?

File: doc/integrator/headers.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Thes vars follows the folowing pattern ``content_security_policy_<path>_<directive>[_extra]``.
```
Comment (jwkaltz) [inline doc/integrator/headers.rst]: The naming of these variables follows this pattern: ``content_security_policy_<path>_<directive>[_extra]``.

File: doc/integrator/headers.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Where ``<path>`` can be: ``main``, ``admin``, ``apihelp``, ``oldapihelp`` or ``c2c``,
+``<directive>`` can be: ``default_src``, ``script_src``, `style_src``, ``img_src``,
+``connect_src`` or ``worker_src``,
+``[_extra]``is a suffix to be able to extend a directive instance of completely overide it.
```
Comment (jwkaltz) [inline doc/integrator/headers.rst]: overide -> overriding

**Comment by sbrunner** [comment]
Thanks @jwkaltz I apply your suggestions with some changes :-)

File: doc/integrator/headers.rst [hunk 6]
```diff
@@ -0,0 +1,28 @@
+.. _integrator_headers:
+
+Headers
+=======
+
+``c2cgeoportal`` add some HTTP headers on its responses with certain default values.
```
Comment (jwkaltz) [inline doc/integrator/headers.rst]: ``c2cgeoportal`` adds

**Pull request merged**